pkg/cmd/roachtest/inconsistency.go

```diff
@@ -22,6 +22,7 @@ func registerInconsistency(r *testRegistry) {
 	r.Add(testSpec{
 		Name:       fmt.Sprintf("inconsistency"),
 		Owner:      OwnerKV,
+		Skip:       "Uses RocksDB put command; unskip when that's bypassed",
```

Comment:
TODO(adityamaru): https://github.com/cockroachdb/cockroach/commit/8732d4281a7f913c53a5a069932d9caa222dcd57#diff-d0816e2d21e74d24bc9c81ef3e8b14ebf0871e73bdb4045dfbbf6e4cea69f920 was merged after this PR picked its roachtest.

Reply:
oh nevermind, 20.1 still uses rocksdb. Nothing to do here.